### PR TITLE
 Remove (# ebooks) from search results 

### PR DESCRIPTION
**Pull request by salman-bhai**
Meant to address #775 
internetarchive

The number of e-books option is removed as requested in the Issue!

### CODE REVIEW

**Comment by salman-bhai** [comment]
## Note:

This is my first PR to the repository and the organisation. Please let me know in case I miss something!

File: solr-update.offset [hunk 1]
```diff
@@ -0,0 +1 @@
+2018-01-30:1633
```
Comment (mekarpeles) [inline solr-update.offset]: I don't think we want to checkin this solr-update.offset file :)

File: openlibrary/macros/SearchResultsWork.html [hunk 2]
```diff
@@ -34,8 +34,6 @@ <h3 class="booktitle">
       <span class="resultPublisher">
         $if doc.edition_count:
           $commify(doc.edition_count) edition$("s" if doc.edition_count != 1 else "")
-          $if len(doc.ia):
```
Comment (mekarpeles) [inline openlibrary/macros/SearchResultsWork.html]: Great! Looks good to me

**Comment by mekarpeles** [comment]
lgtm! thank you!

**Pull request merged**